benchmarks/common.py

```diff
@@ -48,6 +49,26 @@
 output_filename = None
 
 
+# We are primarily interested in tf32 datatype
+torch.backends.cuda.matmul.allow_tf32 = True
```

Comment:
We might want to set this inside `main()`.  A lot of the benchmarks change global state when we import them.

Reply:
Good idea.